Blockzilla/IntroViewController.swift

```diff
@@ -122,7 +122,7 @@ extension IntroViewController: ScrollViewControllerDelegate {
     }
     
     func scrollViewController(scrollViewController: ScrollViewController, didUpdatePageIndex index: Int) {
-        Telemetry.default.recordEvent(category: TelemetryEventCategory.firstRun, method: TelemetryEventMethod.showSlide, object: TelemetryEventObject.onboarding, value: String(index))
+        Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.show, object: TelemetryEventObject.onboarding, value: String(index))
```

Comment:
If this is a new probe, it's fine to change now since we haven't released it yet.